serving/docs/configurations_model.md

```diff
@@ -72,8 +72,6 @@ option.enable_venv=true
 option.rolling_batch=auto
 #option.rolling_batch=lmi-dist
 option.max_rolling_batch_size=64
-option.paged_attention=false
-option.max_rolling_batch_prefill_tokens=1088
```

Comment:
I thought we still allow users to configure `max_rolling_batch_prefill_tokens`?

Reply:
Yes, user can set max_rolling_batch_prefill_tokens, but it's deprecated. Normally user don't need to override this. So I keep this option in the parameter doc and remove it in the example.